### PR TITLE
Upgrade to Play 2.7.0-RC8 & other changes

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: test/test/HelloFunctionalTest.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package test;
 
 import akka.grpc.GrpcClientSettings;
-import akka.grpc.play.JavaAkkaGrpcClientHelpers;
```
Comment (ignasi35) [inline test/test/HelloFunctionalTest.java]: not part of this PR but I find using `test` as package name confusing.
(note how the file is on the `test/test/` folder)

Use `com.example` instead?

**Review by ignasi35 — APPROVED** [review]

File: test/test/HelloFunctionalTest.java (hunk 1 above)
Comment (dwijnand) [reply test/test/HelloFunctionalTest.java:4]: Yeah, we can move them.

Comment (ignasi35) [reply test/test/HelloFunctionalTest.java:4]: raised https://github.com/playframework/akka-grpc-play-quickstart-java/issues/22
